README.md

```diff
@@ -7,133 +7,49 @@ The first "data-first" and "data-last" utility library designed especially for T
 [![NPM](https://img.shields.io/npm/v/remeda)](https://www.npmjs.org/package/remeda)
 ![Dependencies](https://img.shields.io/librariesio/release/npm/remeda)
 
-## Installation
+## Documentation
 
-```bash
-npm i remeda@beta
-yarn add remeda@beta
-```
+Full documentation on [remedajs.com/docs](https://remedajs.com/docs).
 
-Then in .js or .ts
+For Lodash and Ramda users, check the function mapping on [remedajs.com/mapping](https://remedajs.com/mapping).
 
-```js
-import * as R from "remeda"; // tree-shaking supported!
-```
+## Features
 
-## Why Remeda?
+* First-class TypeScript support, with types that are as specific as possible. 
+* Supports data-first (`R.filter(array, fn)`) and data-last (`R.filter(fn)(array)`) approaches.
+* Lazy evaluation support with `pipe` and `piped`.
+* Runtime and types are both extensively tested, with full code coverage.
+* Tree-shakable, supports CJS and ESM.
+* Fully documented with JSDoc, supports in-editor function documentation.
 
-There are no good utility libraries that work well with TypeScript. When working with Lodash or Ramda you must sometimes annotate types manually.
-Remeda is written and tested in TypeScript and that means there won't be any problems with custom typings.
+## Getting started
 
-## What's "data-first" and "data-last"?
-
-Functional programming is nice, and it makes the code more readable. However there are situations where you don't need "pipes", and you want to call just a single function.
-
-```js
-// Remeda
-R.pick(obj, ["firstName", "lastName"]);
+### Installation
 
-// Ramda
-R.pick(["firstName", "lastName"], obj);
-
-// Lodash
-_.pick(obj, ["firstName", "lastName"]);
+```bash
+npm install remeda
+yarn add remeda
 ```
 
-> For readers looking for data-last forms like `R.filter(fn)(array)`, Remeda supports it. Keep reading along!
-
-In the above example, "data-first" approach is more natural and more programmer friendly because when you type the second argument, you get the auto-complete from IDE. It's not possible to get the auto-complete in Ramda because the data argument is not provided.
-
-"data-last" approach is helpful when writing data transformations aka pipes.
+### Usage
 
 ```js
-const users = [
-  { name: "john", age: 20, gender: "m" },
-  { name: "marry", age: 22, gender: "f" },
-  { name: "samara", age: 24, gender: "f" },
-  { name: "paula", age: 24, gender: "f" },
-  { name: "bill", age: 33, gender: "m" },
-];
-
-// Remeda
-R.pipe(
-  users,
-  R.filter((x) => x.gender === "f"),
-  R.groupBy((x) => x.age),
-);
+import * as R from "remeda";
```

Comment:
Let's also mention that imports could be done by name. I think some people might find this syntax confusing.

```ts
import { map } from "remeda";
```